src/main/kotlin/me/modmuss50/mpp/platforms/discord/DiscordWebhookTask.kt

```diff
@@ -61,6 +88,10 @@ abstract class DiscordWebhookTask : DefaultTask(), DiscordWebhookOptions {
     @get:Inject
     protected abstract val workerExecutor: WorkerExecutor
 
+    @JvmField
+    @Suppress("MemberVisibilityCanBePrivate")
+    protected val style: MessageStyle = MessageStyle()
```

Comment:
This isnt right, this should be a `@Nested Property<MessageStyle>`

Reply:
I wasn't able to get it working with `@Nested`, this was the best I was able to do